ext/opcache/ZendAccelerator.c

```diff
@@ -273,7 +277,7 @@ static ZEND_INI_MH(accel_include_path_on_modify)
 
 static inline void accel_restart_enter(void)
 {
-#ifdef ZEND_WIN32
+#if defined(HAVE_STDATOMIC_H) || defined(ZEND_WIN32)
```

Comment:
Check for the given macro instead? That would avoid adjust all conditions again when support for the macro changes.

Reply:
Not my taste ... but I have changed it like you said.